mne_bids/pick.py

```diff
@@ -64,5 +67,4 @@ def coil_type(info, idx):
     for key, values in get_coil_types().items():
         if ch['coil_type'] in values:
             return key
-    raise ValueError('Unknown coil of type {0} '
-                     'for channel {1}'.format(ch['coil_type'], ch["ch_name"]))
```

Comment:
might be worth it to log a warning here.

Reply:
not sure. This is really just to get a more specific channel type.